visualization/requirements.txt

```diff
@@ -0,0 +1,3 @@
+plotly 
+sklearn
+pandas
```

Comment:
Is `pandas` really used, if so, where?

Reply:
`visualization/plot_study.py` requires pandas:
```
| Traceback (most recent call last):
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/optuna/_optimize.py", line 216, in _run_trial
|     value_or_values = func(trial)
|   File "visualization/plot_study.py", line 39, in objective
|     fmnist = fetch_openml(name="Fashion-MNIST", version=1)
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/utils/validation.py", line 63, in inner_f
|     return f(*args, **kwargs)
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 921, in fetch_openml
|     "md5_checksum"])
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 638, in _download_data_to_bunch
|     md5_checksum=md5_checksum)
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 69, in wrapper
|     return f(*args, **kw)
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 518, in _load_arff_response
|     parsed_arff = parse_arff(arff)
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/datasets/_openml.py", line 332, in _convert_arff_data_dataframe
|     pd = check_pandas_support('fetch_openml with as_frame=True')
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/utils/__init__.py", line 1095, in check_pandas_support
|     ) from e
| ImportError: fetch_openml with as_frame=True requires pandas.
| Traceback (most recent call last):
|   File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/sklearn/utils/__init__.py", line 1090, in check_pandas_support
|     import pandas  # noqa
| ModuleNotFoundError: No module named 'pandas'
```

Will add to requirement.txt.